test/model/content_test.dart

```diff
@@ -301,6 +301,21 @@ class ContentExample {
         '\n</code></pre></div>'),
     ]);
 
+  static const lineBreaksAfterCodeBlocks = ContentExample(
```

Comment:
nit: let's name this starting with `codeBlock`, to help group it with the other examples about code blocks